lib/autoupdate/agent/updater.go

```diff
@@ -953,15 +974,18 @@ func (u *Updater) notices(ctx context.Context) error {
 	}
 	if !enabled && active {
 		u.Log.WarnContext(ctx, "Teleport is installed and started, but not configured to start on boot.")
-		u.Log.WarnContext(ctx, "After configuring teleport.yaml, you can enable it with: systemctl enable teleport")
+		u.Log.WarnContext(ctx, "After configuring teleport.yaml, you must enable it.",
+			"command", fmt.Sprintf("systemctl enable %s", u.TeleportServiceName))
```

Comment:
nit: fmt.Sprintf is overkill for simple concatenation: `"systemctl enable " + u.TeleportServiceName`